Hr update (#1)
nhs-r-community/git_training: README.md

# git_training

> Git training from the NHS-R community training steps:

Reviewer: Are these the steps people need to follow before the training?

1. Download [R](https://cran.r-project.org/bin/windows/base/) and [RStudio desktop](https://www.rstudio.com/products/rstudio/download/) or create an account on [RStudio cloud](https://login.rstudio.cloud/login?redirect=%2F).

> 2. Install Git via this [linnk](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) if you are using RStudio desktop.

Reviewer: ```suggestion
2. Install Git via this [link](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) if you are using RStudio desktop.
```

Reviewer: Would add here that if you're using RStudio cloud you do not need this step, right?

Author: indeed!

> 3. Use this repository or create a [new repository.](./guides/how_to_create_a_repo.md) 

Reviewer: by 'use', do you mean 'clone'?

> 4. Follow the introduction to Git and version control for R users [guide](./guides/introduction_to_git.md). Alternatively, for Python users see [here.](https://github.com/NHSDigital/rap-community-of-practice/blob/main/development-approach/01_intro-to-git.md).

Reviewer: Slight rephrasing suggestion, to make it clearer that this is a separate guide:
```suggestion
4. Follow the steps in our [guide to git for R users](./guides/introduction_to_git.md) (see [here.](https://github.com/NHSDigital/rap-community-of-practice/blob/main/development-approach/01_intro-to-git.md) for an equivalent guide for Python users).
```

> 5. Use the [Let's Git started guide](https://happygitwithr.com/index.html) for further info on connecting to Git via RStudio, troubleshooting tips and Git/R features.

Reviewer: I'd flip this, and take it out of the list. I've also removed the name, as this is a little confusing as it's not where we want people to start!
```suggestion

For more information about connecting to Git via RStudio, troubleshooting tips and Git/R features, we'd recommend [this guide](https://happygitwithr.com/index.html) .
```

Author: nice one, thanks

> ## Practice 

Reviewer: Maybe we could rename this section 'What's next?', and include the line above, and also this bit about practice.

Author: Love this

# git_training
Git training from the NHS-R community

## Training guides

> 1. For an introduction to Git and version control for R users see [guide](./guides/intro_to_git.md). For Python users see [guide](https://github.com/NHSDigital/rap-community-of-practice/blob/main/development-approach/01_intro-to-git.md).

Reviewer: I would maybe say something like - this repo contains: 1,2,
For Python users- you can find guidance on the NHS-D CoP.

2. Learn [how to create a repository on GitHub](./guides/how_to_create_a_repo.md).

> ## Exercises 

Reviewer: Can you explain what the exercises are? I think this is a little confusing.

Author: might rename the folder to practice scripts or something similar?

Feel free to practice version control by making changes to the files in the Exercises folder (add, remove or edit files) and then follow the Git commands workflow to push these changes to your branch for this repository on GitHub. 

> Don't forget to follow the Introduction to Git and version control guide mentioned in step 1 above!

Reviewer: I don't know whether it's clear that the above are 'steps' that you should follow- I thought this was more of an explanation of what was in the repo.
If that's not the case, I think that should be explicit.

Author: yeah, I can see that, agreed. Thanks
